runtime/compiler/control/JITaaSCompilationThread.hpp

```diff
@@ -36,6 +36,7 @@ struct TR_RemoteROMStringKey;
 using IPTable_t = PersistentUnorderedMap<uint32_t, TR_IPBytecodeHashTableEntry*>;
 using IPTableHeapEntry = UnorderedMap<uint32_t, TR_IPBytecodeHashTableEntry*>;
 using IPTableHeap_t = UnorderedMap<J9Method *, IPTableHeapEntry *>;
+using resolvedMirrorMethodsPersistIP_t = Vector<TR_ResolvedJ9Method *>;
```

Comment:
Typically data structures start with a capital letter so let's make that `ResolvedMirrorMethodsPersistIP_t`